py3status/modules/velib_metropole.py

```diff
@@ -262,25 +282,27 @@ def velib_metropole(self):
                 ),
                 "index": self.station_index,
             }
-            full_text = self.py3.safe_format(self.format, velib_data)
+
+        self.first_start = False
 
         return {
-            "cached_until": self.py3.time_in(self.cache_timeout),
-            "full_text": full_text,
+            "cached_until": self.py3.time_in(cached_until),
+            "full_text": self.py3.safe_format(self.format, velib_data),
         }
 
     def on_click(self, event):
         button = event["button"]
         if self.stations:
-            self.refresh = False
             if button == self.button_next:
                 self.station_index += 1
                 self.station_index %= self.number_of_stations + 1
             elif button == self.button_previous:
                 self.station_index -= 1
                 self.station_index %= self.number_of_stations + 1
-        elif button == self.button_refresh:
-            self.refresh = True
+        elif button == 2:
```

Comment:
Cool beans. Use `self.button_refresh` and keep `2` in `post_config_hook` with others.